public/hooks/__tests__/use_conversations.test.ts

```diff
@@ -7,17 +7,36 @@ import { useDeleteConversation, usePatchConversation } from '../use_conversation
 import { renderHook, act } from '@testing-library/react-hooks';
 import { useCore } from '../../contexts/core_context';
 import { HttpHandler } from '../../../../../src/core/public';
-
+import { DataSourceServiceMock } from '../../services/data_source_service.mock';
+import { httpServiceMock } from '../../../../../src/core/public/mocks';
 jest.mock('../../contexts/core_context');
-const useCoreMocked = useCore as jest.MockedFunction<typeof useCore>;
+
+const setup = () => {
```

Comment:
There is already one mock file for core_context in `public/contexts/__mocks__/core_context.tsx`, can we add dataSource mock to this mock? So I think we don't need this setup function. We can use the `useCoreMocked` directly.